lib/commands/simctl.js

```diff
@@ -0,0 +1,71 @@
+import { errors } from 'appium/driver';
+
+/**
+ * List of subcommands which accept 'device' target.
+ */
+const SUBCOMMANDS_HAS_DEVICE = [
+  'addmedia',
+  'boot',
+  'get_app_container',
+  'getenv',
+  'icloud_sync',
+  'install',
+  'install_app_data',
+  'io',
+  'keychain',
+  'launch',
+  'location',
+  'logverbose',
+  'openurl',
+  'pbcopy',
+  'pbpaste',
+  'privacy',
+  'push',
+  'shutdown',
+  'spawn',
+  'status_bar',
+  'terminate',
+  'ui',
+  'uninstall'
+];
+
+const commands = {
+
+  /**
```

Comment:
Please move all type defintions to the very end of the module. This also enables them for export.